src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -115,7 +120,8 @@ public void UsingUpdate2()
```

Comment:
Nit: I prefer the following formatting since it's easier to read the embedded code:

```C#
   var src1 = @"
partial record C { }
partial record C(int X);
";
```